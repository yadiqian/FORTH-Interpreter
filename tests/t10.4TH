-3.3 2 4 88 6 54 30 15.7 5.5 1.9 + - * / ^ - - ^ * .