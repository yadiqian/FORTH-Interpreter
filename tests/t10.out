-4.7421614e-25